packages/error/index.ts

```diff
@@ -1,6 +1,16 @@
 const suffix = "\nVisit https://community.frontity.org for help! 🙂\n";
 
-export const error = (message: string, doThrow = true) => {
+interface FrontityError {
+  (
+    message: string,
+    options?: {
+      throw?: boolean;
+    }
+  ): void;
+}
+
+export const error: FrontityError = (message, options = {}) => {
+  const doThrow = typeof options.throw !== "undefined" ? options.throw : true;
   if (process.env.NODE_ENV !== "production") {
     if (doThrow) throw new Error(message + suffix);
     console.error(message + suffix);
```

Comment:
@luisherranz Perhaps there is a reason I don't see but why are we throwing an error and also `console.error()`ing the message?

Reply:
`throw` is like `return`, the function won't run further than that so no console.erro :)

Reply:
sorry I forgot how to javascript for a moment... 🤦‍♂

Reply:
haha no problem